plugin/src/main/java/org/wildfly/swarm/plugin/maven/RepositorySystemSessionWrapper.java

```diff
@@ -175,4 +190,24 @@ public RepositoryCache getCache() {
     private RepositorySystemSession delegate;
 
     private DependencyGraphTransformer transformer;
+
+    private boolean excludeSwarm;
+
+    public class SwarmExcludedTraverser implements DependencyTraverser
+    {
+
+       @Override
+       public boolean traverseDependency(Dependency dependency)
+       {
+          return !SWARM_GROUP_ID.equals(dependency.getArtifact().getGroupId());
```

Comment:
I'll ponder, but if the resolved file artifact isn't available at this phase, then we probably have to do yet-another-phase.  In DependencyManager (called from BuildTool), for instance, I do the full transitive solution in phase 1.  Then I remove any dependency that contains fraction-manifest.yml and perform the full resolution again in phase 2.

That ultimately helps me (probably poorly, given this issue) know which set of artifacts are brought in purely because of fractions, and which might've been asked for directly by the project by subtracting one solution from another.

Once there's another dependency though, that transitively brings in fractions, obviously we have a problem.